integration/tests/ldap_hab_user.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+#shellcheck disable=SC2034
+test_name="ldap_hab_user"
+test_backup_restore=true
+
+do_setup() {
```

Comment:
Would it be helpful to add some assertion to this test that the hab user definitely doesn't exist in /etc/passwd?

Reply:
i put it in the other pr